HTML-CSS/Tasarım ödevi/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+    background-color: rgb(61, 61, 61);
+}
+
+.grid {
+    display: grid; 
+ 
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    
+    grid-template-areas: 
+     'label0 label2 label2 label3 label3'
+     'label0 label4 label5 label5 label6'
+     'label1 label4 label5 label5 label6'
+     'label1 label8 label8 label7 label6';
+    
+    gap: 2vh;
+    height: 98vh;
+    width: 200vh;
```

Comment:
**width: 200vh** kodu pek mantıklı gelmiyor bana. Viewport unit'lerinin tanımlarına göre vh (viewport height) ünitesinin width içerisinde kullanılması pek mantıklı değil gibi. Tabii ki de bu kişisel bir öneri ve araştırdığım hiçbir makalede yaptığın şeyin yanlış olduğunu söyleyen bir şeye rastlayamadım, dolayısıyla büyük bir sorun yok diyebiliriz. 

Aynı zamanda her şey için viewport ünitelerini kullanmak pek doğru değil, aşağıda bunu açıklayan bir Stackoverflow sorusu bıraktım.

Viewport Üniteleri Hakkında Stackoverflow Sorusu: https://stackoverflow.com/questions/60118601/official-docs-of-viewport-width-height-being-bad-practice-in-css